### ISSUE TITLE
For toast notification, the style is broken when the length of sub title is too large

### ISSUE THREAD

**Issue by shyfruan** [carbon-design-system/carbon-components-react#118]

**Comment by hellobrian** [comment]
Can you add a screenshot?

**Comment by shyfruan** [comment]
![image](https://user-images.githubusercontent.com/29720673/28552112-088fc420-711f-11e7-8856-91e482fba383.png)

**Comment by tw15egan** [comment]
Does this occur if that string is not one continuous word? aka if the string was `subtitle test test test testtest test test test test test test test test test test testtest test test testtest test test test`

**Comment by hellobrian** [comment]
Can you also let us know which version of `carbon-components-react` you're using? 
I tried replicating your bug on [react.carbondesignsystem.com](http://react.carbondesignsystem.com/?selectedKind=Notifications&selectedStory=toast&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel) but the text doesn't break through the notification like you show. 

![image](https://user-images.githubusercontent.com/4185382/28578709-21723f26-7120-11e7-87f2-33b812a575ee.png)

Is it possible there's an overriding style somewhere in your project that is not allowing the subtitle to stay inside the notification?

**Comment by shyfruan** [comment]
We have used `carbon-components-react version` 3.4.4. So 3.5.1 has solved the problem?

**Comment by hellobrian** [comment]
I'm seeing the issue now. 
One way to fix this is to override the `width` styles on the notification. 
Right now, it's set at a fixed width (`width: 16.875rem;`). You can override it like this:

```jsx
<Notification
  style={{width: '100%'}}
  kind="error"
  caption="hi"
  title="title"
  subtitle="subtitle_but_imagine_this_is_a_really_long_string_with_no_whitespace"
/>
```

![image](https://user-images.githubusercontent.com/4185382/28675629-0f2ea388-72ae-11e7-90d5-a11d0047e1d2.png)

☝️ But...this makes the Notification way too wide... 

What we should do instead is force the long string to break based on the fixed width, so we need to update the SCSS in [carbon-components](https://github.com/carbon-design-system/carbon-components). So, here's what the Notification could look like if we allow the long strings to break:

![image](https://user-images.githubusercontent.com/4185382/28675870-c3bbd0c8-72ae-11e7-83a2-a71e9bc943e0.png)

__Resource:__ [Handling Long Words and URLs (Forcing Breaks, Hyphenation, Ellipsis, etc) | CSS-Tricks](https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/)

**Comment by hellobrian** [comment]
Issue moved to [carbon-design-system/carbon-components #238](https://github.com/carbon-design-system/carbon-components/issues/238) via [**ZenHub**](https://www.zenhub.com/)
